docker/Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN cd /webapp-frontend-deps/ && npm install
 # Install Socorro Python requirements
 COPY --chown=app:app ./requirements /app/requirements
 RUN pip install -U 'pip>=8' && \
-    pip install --no-cache-dir -r requirements/default.txt -c requirements/constraints.txt && \
+    pip install --use-feature=2020-resolver --no-cache-dir -r requirements/default.txt -c requirements/constraints.txt && \
```

Comment:
Oops--accidentally pushed something I didn't intend to that changed the resolver pip used when building.